src/jit/jitconfigvalues.h

```diff
@@ -197,6 +210,7 @@ CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 1) // Controls the Alt
 
 #if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
 CONFIG_INTEGER(EnableSSE3_4, W("EnableSSE3_4"), 1) // Enable SSE3, SSSE3, SSE 4.1 and 4.2 instruction set as default
+JIT_CONFIG_DWORD_INFO(EnableSSE3_4, W("EnableSSE3_4"), 1, "Enable SSE3, SSSE3, and SSE4 instruction set as default")
```

Comment:
Macro `JIT_CONFIG_DWORD_INFO` is used to generate the doc. Because most of the JIT config values are already included in `clrconfigvalues.h`, here we only need to generate doc for new JIT config values that are not in `clrconfigvalues.h`, so I think a new macro is necessary.